com.github.jean28518.Linux-Assistant.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.github.jean28518.Linux-Assistant
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+command: linux-assistant
+finish-args:
+  - "--socket=fallback-x11"
+  - "--device=dri"
+  - "--share=ipc"
+  - "--filesystem=host"
+  - "--talk-name=org.freedesktop.Flatpak"
```

Comment:
this is not allowed by default.

Reply:
Linux-Assistant needs right out of the start this feature for recoginizing the host behind it and to run some administrive tasks.

Is there an alternative?

Reply:
Done from my side. Linux-Assistant isn't using --talk-name=org.freedesktop.Flatpak anymore on build and install time.